boards/ct/ctcc_nrf52840/ctcc_nrf52840_defconfig

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable hardware stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# NOTE: if a device is used with an adapter/dongle to USB,
+# reset signal should be disabled due to missing on-board pullup.
+# In case of plugging a device to a mPCIe/M.2 slot,
+# the reset signal might be handled by the host device
+# and the GPIO_AS_PINRESET can be set to 'y'
+CONFIG_GPIO_AS_PINRESET=n
```

Comment:
this is deprecated, use the `gpio-as-nreset` dt property

Reply:
thanks for pointing it out, removed